anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -658,3 +665,87 @@ def transform(self, frames, show_progress=False, normalize=True, rust_moments=Tr
             return normalized_features
         else:
             return features
+
+    def power_spectrum(self, ell_frames, AniSOAP_HYPERS=None, sum_over_samples=True):
+        """Function to compute the power spectrum of AniSOAP
+
+        computes the power spectrum of AniSOAP with the inputs of AniSOAP hyperparameters
+        and ellipsoidal frames using ellipsoidal density projection. It checks if
+        each ellipsoidal frame contains all required attributes and processes
+        Clebsch-Gordan coefficients for the angular component of the AniSOAP descriptors.
+
+        Parameters
+        ----------
+
+        ell_frames: list
+        a list of ellipsoidal frames, where each frame contains attributes:
+        'c_diameter[1]', 'c_diameter[2]', 'c_diameter[3]', 'c_q', 'positions', and 'numbers'.
+        It only accepts c_q for the angular attribute of each frame.
+
+        AniSOAP_HYPERS : keyword arguments
+        A dictionary of hyperparameters for the power spectrum calculation, including 'max_angular'
+        'max_radial', 'radial_basis_name', 'rotation_type', 'rotation_key', 'cutoff_radius', 'radial_gaussian_width',
+        'basis_rcond', and 'basis_tol'.
+
+        sum_over_sample: bool
+        A function that returns the sum of coefficients of the frames in the sample.
+
+        Returns
+        -------
+        x_asoap_raw, a power spectrum of AniSOAP descriptor that can be visualized using jupyter notebook.
```

Comment:
Update this description, since it can either return `x_asoap_raw` or `mvg_nu2`
It's important to specify the types, that `x_asoap_raw` is a `np.ndarray`, and `mvg_nu2` is a `TensorMap`